typesystem/fields.py

```diff
@@ -1,4 +1,6 @@
 import decimal
+import imghdr
```

Comment:
Deprecated since 3.11.
https://peps.python.org/pep-0594/#imghdr

Reply:
typesystem support version <= 3.10

Reply:
3.11 releases just in one week. The typesystem will support it without any doubt.

Reply:
Additional dependency lib is not cost-effective. Or do not check the image type.

such as : `filetype` OR `puremagic` in [pypi](https://peps.python.org/pep-0594/#imghdr)

Reply:
This can be an optional dependency for those, who wants to use File fields.

Reply:
Maybe `Image` is unnecessary.  Just define the `File`, right?

Reply:
It would be nice if `File()` can validate the file type. So we can specify that we want image, audio, video, etc.
Something like 
```python
File(accept=['image/jpeg', 'image/png'])
```
This should answer your question. But IMO, a dedicated `Image()` field is a nice utility.